sections/builder.js

```diff
@@ -283,15 +283,21 @@ export default [
   {
     type: "method",
     method: "with",
-    example: ".with(alias, function|raw)",
-    description: "Add a \"with\" clause to the query. \"With\" clauses are supported by PostgreSQL, Oracle, SQLite3 and MSSQL.",
+    example: ".with(alias, [*columns], callback|builder|raw)",
```

Comment:
just `columns` should be fine, if it is a mandatory array param

Reply:
Both with(alias, query) and with(alias, columns, query) are valid. It sounds like [columns] would be the way to write it though, without the star?

Reply:
yup, fair